app/src/main/java/org/simple/clinic/drugs/selectionv2/PrescribedDosageAdapter.kt

```diff
@@ -0,0 +1,64 @@
+package org.simple.clinic.drugs.selectionv2
+
+import android.support.v7.recyclerview.extensions.ListAdapter
+import android.support.v7.util.DiffUtil
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import kotterknife.bindView
+import org.simple.clinic.R
+import org.simple.clinic.drugs.DosageType
+import javax.inject.Inject
+
+class PrescribedDosageAdapter @Inject constructor() : ListAdapter<PrescribedDosageListItem, PrescribedDosageViewHolder>(PrescribedDosageDiffer()) {
+
+  override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): PrescribedDosageViewHolder {
+    val layout = LayoutInflater.from(parent.context).inflate(R.layout.prescribed_drug_with_dosage_list_item, parent, false)
+    return PrescribedDosageViewHolder(layout)
+  }
+
+  override fun onBindViewHolder(holder: PrescribedDosageViewHolder, position: Int) {
+    holder.dosageItem = getItem(position)
+    holder.render()
+  }
+}
+
+class PrescribedDosageViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+
+  private val dosageTextView by bindView<TextView>(R.id.prescribed_drug_list_item_dosage_name)
+  private val divider by bindView<View>(R.id.prescribed_drug_list_item_divider)
```

Comment:
That way every UI element is a `View`. I did not suffix this one because of that. It is understood.
Also, it is being used as both a `divider` and a `dividerView` elsewhere.